.github/workflows/run-deploy.yaml

```
@@ -110,12 +110,16 @@ jobs:
         name: Get Lightsail Instance IP Address
         run: echo ::set-output name=IP_ADDRESS::$(aws lightsail get-instance-access-details --instance-name project-lab-${{ steps.get-branch.outputs.branch }} | jq -r '.accessDetails.ipAddress')
 
+      - id: get-lightsail-ssh-key
+        name: Get Lightsail Private SSH Key
+        run: mkdir ~/.ssh && aws lightsail download-default-key-pair | jq '.privateKeyBase64' | tr -d '"' | sed 's/\\n/\n/g' > ~/.ssh/lightsail.pem
```

I suggest saving the private key in an [output parameter](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter) instead of a file:
```suggestion
        run: echo ::set-output name=SSH_PRIVATE_KEY::$aws lightsail download-default-key-pair | jq -r '.privateKeyBase64' | head -n -1)
```
_NOTE: I changed the command because I realized it could be shortened even more by using the `-r` flag in `jq` to get rid for the need of using the `tr` and `sed` commands. The `head -n -1` will discard the last line output by `jq`, which is just an empty line and is not needed_

That way, you can reference it in the `key` and `ssh_private_key` parameter for the SSH and SFTP actions like this:
```yml
key: ${{ steps.get-lightsail-ssh-key.outputs.SSH_PRIVATE_KEY }}
```

---

Thanks a lot!!! Sounds great. I will add this to the PR